plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -421,92 +421,45 @@ func TestPodFitsResources(t *testing.T) {
 	}
 
 	storagePodsTests := []struct {
-		pod           *v1.Pod
-		emptyDirLimit int64
-		storageMedium v1.StorageMedium
-		nodeInfo      *schedulercache.NodeInfo
-		fits          bool
-		test          string
-		reasons       []algorithm.PredicateFailureReason
+		pod      *v1.Pod
+		nodeInfo *schedulercache.NodeInfo
+		fits     bool
+		test     string
+		reasons  []algorithm.PredicateFailureReason
 	}{
 		{
-			pod: newResourcePod(schedulercache.Resource{MilliCPU: 1, Memory: 1, StorageOverlay: 1}),
+			pod: newResourcePod(schedulercache.Resource{MilliCPU: 1, Memory: 1}),
 			nodeInfo: schedulercache.NewNodeInfo(
-				newResourcePod(schedulercache.Resource{MilliCPU: 10, Memory: 10, StorageOverlay: 20})),
+				newResourcePod(schedulercache.Resource{MilliCPU: 10, Memory: 10})),
 			fits: false,
 			test: "due to container scratch disk",
 			reasons: []algorithm.PredicateFailureReason{
 				NewInsufficientResourceError(v1.ResourceCPU, 1, 10, 10),
-				NewInsufficientResourceError(v1.ResourceStorageScratch, 1, 20, 20),
 			},
 		},
 		{
-			pod: newResourcePod(schedulercache.Resource{MilliCPU: 1, Memory: 1, StorageOverlay: 10}),
+			pod: newResourcePod(schedulercache.Resource{MilliCPU: 1, Memory: 1}),
 			nodeInfo: schedulercache.NewNodeInfo(
 				newResourcePod(schedulercache.Resource{MilliCPU: 2, Memory: 10})),
 			fits: true,
 			test: "pod fit",
 		},
 		{
-			pod: newResourcePod(schedulercache.Resource{MilliCPU: 1, Memory: 1, StorageOverlay: 18}),
-			nodeInfo: schedulercache.NewNodeInfo(
-				newResourcePod(schedulercache.Resource{MilliCPU: 2, Memory: 2, StorageOverlay: 5})),
-			fits: false,
-			test: "request exceeds allocatable overlay storage resource",
-			reasons: []algorithm.PredicateFailureReason{
-				NewInsufficientResourceError(v1.ResourceStorageScratch, 18, 5, 20),
-			},
-		},
-		{
-			pod: newResourcePod(schedulercache.Resource{StorageOverlay: 18}),
-			nodeInfo: schedulercache.NewNodeInfo(
-				newResourcePod(schedulercache.Resource{MilliCPU: 2, Memory: 2, StorageOverlay: 5})),
-			fits: false,
-			test: "request exceeds allocatable overlay storage resource",
-			reasons: []algorithm.PredicateFailureReason{
-				NewInsufficientResourceError(v1.ResourceStorageScratch, 18, 5, 20),
-			},
-		},
-		{
-			pod:           newResourcePod(schedulercache.Resource{MilliCPU: 1, Memory: 1, StorageOverlay: 10}),
-			emptyDirLimit: 15,
-			storageMedium: v1.StorageMediumDefault,
+			pod: newResourcePod(schedulercache.Resource{EphemeralStorage: 25}),
```

Comment:
I think you can still keep most of the tests with some modification. Just change overlay to ephemeralstorage with some modification. This could also help us verify the changes related to emptyDir.